Helpers.js

```diff
@@ -0,0 +1,223 @@
+/**
+ * Add a toast on the bottom right corner indicating status
+ */
+function toast(title="Message", description="Description"){
```

Comment:
Not sure what the purpose of the default values for the parameters is. Calling this with just a title seems reasonable, but that would result in a toast with "Description" as a description, which isn't great.

Reply:
Indeed, that was to test the function from the Apps Script UI, I changed it to a Demo function instead and removed the parameters: 

```
/**
 * Add a toast on the bottom right corner indicating status
 */
function toast(title, description){
  SpreadsheetApp.getActive().toast(description, title, -1);
}

/**
 * Shows a demo toast in the UI to demonstrate how the modal looks
 */
function toastDemo(){
  toast("Message","Description")
}
```